.dockerignore

```diff
@@ -2,3 +2,17 @@
 deps/*
 doc
 ebin
+AUTHORS
+CONTRIBUTING
+CONTRIBUTORS
+build.cmd
+Dockerfile
+Dockerfile.*
+LICENSE
+Makefile
+prepare-release.sh
+*.md
+start.*
+TRANSLATORS
+zotonic_install
+priv/emacs
```

Comment:
I disagree. I'm listing here explicitly the files that I *know* don't care about. Any files not listed (that may pop up in future commits and may be necessary for the system) I can't be sure about so I let them be included until such time as their usefulness can be ascertained.